packages/components/src/components/hds/form/super-select/multiple/base.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import PowerSelectComponent from 'ember-power-select/components/power-select';
+import anchoredPositionModifier from '../../../../../modifiers/hds-anchored-position';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+const DEFAULT_HORIZONTAL_POSITION = 'bottom-start';
+const HORIZONTAL_POSITION_MAPPING = {
+  left: 'bottom-start',
+  center: 'bottom',
+  right: 'bottom-end',
+};
+
+export default class HdsSuperSelectMultipleBaseComponent extends PowerSelectComponent {
+  @tracked powerSelectAPI;
+  @tracked showOnlySelected = false;
+  @tracked showNoSelectedMessage = false;
+
+  get selectedCount() {
+    return this.selected?.length || '0';
+  }
+
+  get optionsCount() {
+    return this.options?.length || '0';
+  }
+
+  get resultCountMessage() {
+    return `${this.selectedCount} selected of ${this.optionsCount} total`;
+  }
+
+  @action calculatePosition(trigger, content) {
```

Comment:
@alex-ju here you have access to the `content` which is (I suppose) the dropdown. could you maybe hook here, to apply the `popover` attribute? 🤔

Reply:
Resolving this for now, maybe in the future we will want to explore this option.

Reply:
I made a note to look into this – we could add it in a follow-up PR